tests/core/Jalasoft.TeamUp.Resumes.Core.Tests/ResumesServiceTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿namespace Jalasoft.TeamUp.Resumes.Core.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using Jalasoft.TeamUp.Resumes.DAL.Interfaces;
+    using Jalasoft.TeamUp.Resumes.Models;
+    using Moq;
+    using Xunit;
+
+    public class ResumesServiceTests
```

Comment:
I think the name should have a relationship with the controller name in this case GetResumesServideTests

Reply:
about this let's follow this convention
for API level: GetMyAPI**Tests**
for Core level: ResumesService**Tests**

so the API test will only be concerned about the API layer
and Core tests will only be concerned about Core layer

do you agree?

Reply:
Yes